action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
     description: "Arguments passed to the builder"
     required: true
     default: "--help"
+  cas_token:
```

Comment:
This feels backwards, github actions already have env handling, we should use that.

Reply:
the issue is, we need have it on docker

Reply:
That's not a problem, see <https://github.com/home-assistant/builder/pull/112#discussion_r753100410>

Reply:
nevermind..